vsts/pipelines/templates/_releaseTemplate.yml

```diff
@@ -0,0 +1,120 @@
+    project: '0bdbc590-a062-4c3f-b0f6-9383f67865ee'
+    pipeline: 11021
+    specificBuildWithTriggering: true
+    artifactName: drop
+
+- task: CopyFiles@2
+  displayName: 'Copy Files to: $(Build.ArtifactStagingDirectory)'
+  inputs:
+    SourceFolder: '$(System.ArtifactsDirectory)'
+    TargetFolder: '$(Build.ArtifactStagingDirectory)'
+
+- task: ms-devlabs.utilitytasks.task-Shellpp.Shell++@0
+  displayName: 'Pull and create release tags for build images'
+  inputs:
+    type: FilePath
+    scriptPath: ./vsts/scripts/create-releasetags-buildimage.sh
+  condition: and(succeeded(), eq(variables['ReleaseBuildImages'], 'true'))
+
+- task: ms-devlabs.utilitytasks.task-Shellpp.Shell++@0
+  displayName: 'Pull and create release tags for runtime images'
+  inputs:
+    type: FilePath
+    scriptPath: ./vsts/scripts/create-releasetags-runtimeimages.sh
+  condition: and(succeeded(), eq(variables['ReleaseRuntimeImages'], 'true'))
+
+- task: Docker@0
+  displayName: 'Push Build image to DockerHub'
+  inputs:
+    containerregistrytype: 'Container Registry'
+    dockerRegistryConnection: oryx
+    action: 'Push images'
+    imageNamesPath: '$(Build.ArtifactStagingDirectory)/drop/images/build-images-dockerhub.txt'
+    includeLatestTag: false
+    enforceDockerNamingConvention: true
+  condition: and(succeeded(), eq(variables['ReleaseBuildImages'], 'true'))
+
+- task: Docker@1
+  displayName: 'Push build image to ACR'
+  inputs:
+    azureSubscriptionEndpoint: ${{ parameters.ascName }}
+    azureContainerRegistry: ${{ parameters.acrProdName }}
+    command: 'Push an image'
+    pushMultipleImages: true
+    imageNamesPath: '$(Build.ArtifactStagingDirectory)/drop/images/build-images-mcr.txt'
+    includeLatestTag: false
+    enforceDockerNamingConvention: true
+  condition: and(succeeded(), eq(variables['ReleaseBuildImages'], 'true'))
+
+- task: Docker@0
+  displayName: 'Push runtime images to DockerHub'
+  inputs:
+    containerregistrytype: 'Container Registry'
+    dockerRegistryConnection: oryx
+    action: 'Push images'
+    imageNamesPath: '$(Build.ArtifactStagingDirectory)/drop/images/runtime-images-dockerhub.txt'
+    includeLatestTag: false
+    enforceDockerNamingConvention: false
+  condition: and(succeeded(), eq(variables['ReleaseRuntimeImages'], 'true'))
+
+- task: Docker@1
+  displayName: 'Push runtime images to ACR'
+  inputs:
+    azureSubscriptionEndpoint: ${{ parameters.ascName }}
+    azureContainerRegistry: ${{ parameters.acrProdName }}
+    command: 'Push an image'
+    pushMultipleImages: true
+    imageNamesPath: '$(Build.ArtifactStagingDirectory)/drop/images/runtime-images-mcr.txt'
+    includeLatestTag: false
+    enforceDockerNamingConvention: false
+  condition: and(succeeded(), eq(variables['ReleaseRuntimeImages'], 'true'))
+
+- task: ShellScript@2
+  displayName: 'Clean up Docker containers and images'
+  inputs:
+    scriptPath: ./vsts/scripts/dockerCleanup.sh
+  condition: true
+
+- task: ShellScript@2
```

Comment:
This task should not be required here as we are not running any integration tests

Reply:
removed